packages/@instana-integration/go/README.md

```diff
@@ -44,3 +44,23 @@ Below are the resource attributes that are currently supported by this integrati
 |----------------------------|-------|------------------------|
 | service.name               | string  | This attribute is used to describe the entity name.    |
 | service.instance.id        | string  | This attribute is used to describe the entity ID of the current object.  |
+
+### Installation and Usage
+
+With Instana CLI [stanctl-integration](https://github.com/instana/observability-as-code) for integration package management, you can manage the lifecycle of this package by downloading and importing into Instana.
+
+Downloading the package:
+
+```shell
+$ stanctl-integration download --package @instana-integration/go
+```
+
+Importing the dashboard into Instana:
+
+```shell
+$ stanctl-integration import --package @instana-integration/go \
+  --server $INSTANA_SERVER \
```

Comment:
Do we need explanation about the env vars like $INSTANA_SERVER and $API_TOKEN? Same applies to nodejs and opentelemetry-demo package.

Reply:
I'm not sure, because there are more detailed introductions in the blog and IBM doc, and it's difficult to describe them in concise language here.

Reply:
Added explanations for them, which are derived from their descriptions in Instana openapi doc